From 21f07faea032e8d3b1784fd9f475a972bc59f25d Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 12 Mar 2024 12:32:48 +0800
Subject: [PATCH 04/10] FROM ARCHLINUX: fix(VBox/Additions): disable
 vbglR3GuestCtrlDetectPeekGetCancelSupport
X-Developer-Signature: v=1; a=openpgp-sha256; l=859; i=xtexchooser@duck.com;
 h=from:subject; bh=gtxdVwzeHCqto4QQY3gF/NHQiF3QaVEBNk6oPnwNUhQ=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDGkvVLYY/ne3XdbfuO61z/mnpoebFv6ft3WV3y8WGYuYF
 xHyVxkPdpSyMIhxMciKKbIUGTZ4s+qk84suK5eFmcPKBDKEgYtTACYS18HwT/dalUD1hNkLfh05
 Fql+T53nq/G/zJfJi96/nvAsVybT7xUjw8wn4g9NH/g++sq/xPBfzDOJ6a++bHvittp8wdmM7Q8
 2zeIAAA==
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

---
 .../Additions/common/VBoxGuest/lib/VBoxGuestR3LibGuestCtrl.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibGuestCtrl.cpp b/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibGuestCtrl.cpp
index 0657d9da01f0..2125a121673f 100644
--- a/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibGuestCtrl.cpp
+++ b/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibGuestCtrl.cpp
@@ -222,6 +222,9 @@ static int vbglR3GuestCtrlMsgWaitFor(uint32_t idClient, uint32_t *pidMsg, uint32
  */
 DECL_NO_INLINE(static, bool) vbglR3GuestCtrlDetectPeekGetCancelSupport(uint32_t idClient)
 {
+
+    return false;
+
     /*
      * Seems we get VINF_SUCCESS back from the host if we try unsupported
      * guest control messages, so we need to supply some random message
-- 
2.46.0

